tests/unit/utils/test_misc.py

```diff
@@ -81,3 +82,10 @@ def test_err() -> None:
 
     assert Err(ValueError()).is_ok is False
     assert Err(ValueError()).is_err is True
+
+
+def test_timeit() -> None:
```

Comment:
Maybe extend this a tiny bit to check that nested timers work?

Reply:
done